### PR TITLE
github: config for probot-stale bot

### PR DESCRIPTION
**Pull request by nonsense**
This PR is about introducing the stale bot (https://probot.github.io/apps/stale/) to the go-ethereum repo. It has to be enabled/installed by someone with admin access to the Github organisation.

The idea is to mark as stale all PRs and Issues which are older than one year, and in case there is no activity on them, close them 40 days later.

cc @karalabe @fjl 

In case this is approved (and AFAIK it has already been discussed and approved), then @holiman you would have to click the install and point the bot to the go-ethereum repo, which should just pick-up this file.

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 366
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 40
```
Comment (karalabe) [inline .github/stale.yml]: Why 40? 30 or 42 would be nicer :P

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```
Comment (karalabe) [inline .github/stale.yml]: I'd rather go with `stale` or something similar. `wontfix` seems like a deliberate choice not to fix the issue, whereas here we're just talking about it most probably being not relevant.

File: .github/stale.yml (hunk 1 above)
Comment (nonsense) [reply .github/stale.yml]: Because you said "more than one month" :D 42 it is then.

File: .github/stale.yml (hunk 2 above)
Comment (nonsense) [reply .github/stale.yml]: Changed to `stale`.

**Review by meththot — COMMENTED** [review]
Thanka

**Review by meththot — COMMENTED** [review]
nonsense:probot-stale

**Pull request merged**